features/doctrine/date-filter.feature

```diff
@@ -10,7 +10,7 @@ Feature: Date filter on collections
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json"
-    And the JSON should be equal to:
+    And the JSON should be valid according to this schema:
```

Comment:
The idea of using a JSON Schema instead of raw values is great but I what is the interest here has the schema isn't provided?